dwst/scripts/test/particles.js

```diff
@@ -13,10 +13,23 @@
 
 */
 
-import {parseParticles, escapeForParticles, InvalidParticles} from '../particles.js';
+import {parseParticles, escapeForParticles, InvalidParticles, invalidToMlog, finishInvalidParticle} from '../particles.js';
 import {expect} from 'chai';
 
 describe('particles module', () => {
```

Comment:
Add separate tests for each type of broken markup that are supposed to cause `parseParticles` to throw an error.